packages/client/src/Feed.js

```diff
@@ -1,4 +1,4 @@
-import { useContext, useState, useEffect } from "react";
+import { useContext, useState, useEffect, useCallback } from "react";
```

Comment:
Doesn't look like we're using `useCallback` here. I assume the idea was to wrap `populateFeed` the better suggestion would be to move `populateFeed` into the `useEffect` below.